### PR TITLE
Wolka

### PR DESCRIPTION
**Pull request by antgonza**

### CODE REVIEW

File: qp_shogun/shogun/__init__.py [hunk 1]
```diff
@@ -37,6 +37,8 @@
     'Taxonomic Predictions - phylum': 'BIOM',
     'Taxonomic Predictions - genus': 'BIOM',
     'Taxonomic Predictions - species': 'BIOM',
+    'Woltka - gOTU': 'BIOM',
```
Comment (qiyunzhu) [inline qp_shogun/shogun/__init__.py]: Per our offline discussion, we think that "per_genome" serves the goal more accurately for the moment.

**Review by qiyunzhu — COMMENTED** [review]
Looks good to me! I added one comment inline.

**Review by qiyunzhu — APPROVED** [review]
Good job!

File: qp_shogun/shogun/shogun.py [hunk 2]
```diff
@@ -258,6 +259,28 @@ def shogun(qclient, job_id, parameters, out_dir):
         aname = 'Taxonomic Predictions - %s' % level
         ainfo.append(ArtifactInfo(aname, 'BIOM', [(output, 'biom')]))
 
+    # Wolka only works with WOL databases
```
Comment (ElDeveloper) [inline qp_shogun/shogun/shogun.py]:
```suggestion
    # Woltka only works with WOL databases
```

File: qp_shogun/shogun/tests/test_shogun.py [hunk 3]
```diff
@@ -174,18 +177,36 @@ def test_generate_shogun_dflt_params(self):
                 'Capitalist': False,
                 'Number of threads': 15,
                 'Percent identity': 0.95},
-            'rep82_utree': {
-                'Database': join(self.db_path, 'rep82'),
-                'Aligner tool': 'utree',
+            # 'rep82_utree': {
```
Comment (ElDeveloper) [inline qp_shogun/shogun/tests/test_shogun.py:180]: Can you remove these commented-out lines?

File: qp_shogun/shogun/utils.py [hunk 4]
```diff
@@ -16,9 +16,10 @@
 import pandas as pd
 from biom import Table
 
-ALIGNERS = ["utree",
-            "burst",
-            "bowtie2"]
+ALIGNERS = [
+    # "utree",
```
Comment (ElDeveloper) [inline qp_shogun/shogun/utils.py:20]: Can we remove these commented-out lines?

File: qp_shogun/shogun/tests/test_shogun.py [hunk 5]
```diff
@@ -500,6 +521,69 @@ def test_shogun_bt2(self):
...
+                           'biom')]),
+            ArtifactInfo('Taxonomic Predictions - species', 'BIOM',
+                         [(pout_dir('otu_table.redist.species.biom'),
+                           'biom')]),
+            ArtifactInfo('Woltka - per genome', 'BIOM',
+                         [(pout_dir('woltka_per_genome.biom'), 'biom')]),
+            ArtifactInfo('Woltka - per gene', 'BIOM',
+                         [(pout_dir('woltka_per_gene.biom'), 'biom')])]
+
+        self.assertCountEqual(ainfo, exp)
```
Comment (ElDeveloper) [inline qp_shogun/shogun/tests/test_shogun.py:585]: Is it possible to add tests that check for specific artifact attributes other than the total count of resulting artifacts?

**Review by ElDeveloper — APPROVED** [review]
Looks good, just a few notes.

File: qp_shogun/shogun/tests/test_shogun.py (hunk 3 above)
Comment (antgonza) [reply qp_shogun/shogun/tests/test_shogun.py:180]: I prefer to keep them as there is a chance that in a future release we will need to readd ...

File: qp_shogun/shogun/tests/test_shogun.py (hunk 5 above)
Comment (antgonza) [reply qp_shogun/shogun/tests/test_shogun.py:585]: AFAIK, it already does that: https://bugs.python.org/msg265980

Comment (ElDeveloper) [reply qp_shogun/shogun/tests/test_shogun.py:585]: Didn't know that, thanks!

**Comment by ElDeveloper** [comment]
Thanks @antgonza!

**Pull request merged**